### PR TITLE
Add support for pyinstaller-compressed conda installations

### PR DESCRIPTION
**Pull request by mattgodbolt**

### CODE REVIEW

**Review by cread — APPROVED** [review]
Great idea. Thanks.

File: ozy/installers/conda.py [hunk 1]
```diff
@@ -9,18 +12,36 @@
 
 class CondaInstaller(Installer):
     def __init__(self, name, config):
-        super().__init__(name, config, 'package', 'version', channels=[], conda_bin="conda")
+        super().__init__(name, config, 'package', 'version', channels=[], conda_bin="conda", pyinstaller=False)
```
Comment (kpweiler) [inline ozy/installers/conda.py:15]: Any reason to not make the default True here? I'm having a hard time thinking of downsides to deploying python applications like this in the context of ozy.

**Review by kpweiler — APPROVED** [review]
this is ♠️

File: ozy/installers/conda.py (hunk 1 above)
Comment (mattgodbolt) [reply ozy/installers/conda.py:15]: `conda` and `mamba` both need to not be squashed in this way, and I don't know what else might not work. But mostly it's scaredy-cat-ness on my part!

Comment (cread) [reply ozy/installers/conda.py:15]: I can think of a few reasons not to make the default True here. The primary one being that `pyinstaller` is not always able to detect what needs to be bundled into the file, and so does not always work without help.

**Pull request merged**